SLRemoting/SLObject.m

```diff
@@ -229,7 +229,7 @@ + (NSDictionary *)convertArguments:(NSDictionary *)prop {
         id value = [prop objectForKey:key];
         if ([value isKindOfClass:[NSDate class]]) {
             NSString *jsonString = [value convertToJSONString];
-            value = @{ @"$type": @"date", @"$data": jsonString };
+            value = jsonString ;
```

Comment:
Is there a way how to support both LB2.x and LB3.x servers with a single SDK client? I am concerned that with this change in place, users won't be able to use the latest SDK version with their LB2.x apps.

Also I don't recall changing the format of date arguments in strong-remoting, why is this fix needed in the first place? Is this perhaps a side-effect of my coercion refactor that I did not discover?